string.c

```diff
@@ -3075,23 +3077,20 @@ rb_str_concat_literals(size_t num, const VALUE *strary)
 
 /*
  *  call-seq:
- *     str.concat(obj1, obj2, ...)          -> str
- *
- *  Concatenates the given object(s) to <i>str</i>. If an object is an
- *  Integer, it is considered a codepoint and converted to a character
- *  before concatenation.
+ *     string.concat(*objects) -> new_string
  *
- *  +concat+ can take multiple arguments, and all the arguments are
- *  concatenated in order.
+ *  Returns a new \String containing the concatenation
+ *  of +self+ and all objects in +objects+:
  *
- *     a = "hello "
- *     a.concat("world", 33)      #=> "hello world!"
- *     a                          #=> "hello world!"
+ *    s = 'foo'
+ *    s.concat('bar', 'baz') # => "foobarbaz"
  *
- *     b = "sn"
- *     b.concat("_", b, "_", b)   #=> "sn_sn_sn"
+ *  For each given object +object+ that is an \Integer,
+ *  concatenates the value <tt>object.chr</tt> instead of +object+ itself:
```

Comment:
It's more accurate to state that an integer is treated as a codepoint.  It doesn't actually call chr, as you can test by overriding Integer#chr.  So I think that part should be reverted.  Similar issue with << below.

Reply:
Fixed.